TipCalculator/MainViewController.swift

```diff
@@ -0,0 +1,117 @@
+//
+//  ViewController.swift
+//  TipCalculator
+//
+//  Created by vmalikov on 1/30/16.
+//  Copyright © 2016 justForFun. All rights reserved.
+//
+
+import UIKit
+
+extension Float {
```

Comment:
Move this to a separate file (like FloatExtension, or just Extensions since the project is not large)